app/src/main/java/com/example/mytestproject/ui/searchCity/SearchCityAdapter.kt

```diff
@@ -0,0 +1,66 @@
+package com.example.mytestproject.ui.searchCity
+
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.example.domain.models.CityModel
+import com.example.mytestproject.R
+import com.example.mytestproject.databinding.CellCityBinding
+
+class SearchCityAdapter: RecyclerView.Adapter<SearchCityAdapter.SearchCityViewHolder>() {
+
+    private val cityModelList = mutableListOf<CityModel>()
+
+    private var onclickListenerCityModel: OnClickListenerCityModel? = null
+
+    fun attachCityModel(onclickListener: OnClickListenerCityModel) {
```

Comment:
The name of the method doesn't match the content.

Reply:
Fixed